websites/H/HTB Academy/presence.ts

```diff
@@ -0,0 +1,74 @@
+const presence = new Presence({
+		clientId: "1200517025383075840",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let persred: HTMLElement,
+	module: HTMLElement,
+	persblue: HTMLElement,
+	persgrn: HTMLElement,
+	section: HTMLElement,
+	iterminal: HTMLElement,
+	module2: HTMLElement;
```

Comment:
It doesn't make much sense for these to be global variables considering how they are used